rpc/gnmi/gnmi.proto

```diff
@@ -304,14 +304,18 @@ message SetRequest {
 
 // SetResponse is the response to a SetRequest, sent from the target to the
 // client. It reports the result of the modifications to the data tree that were
-// specified by the client.
+// specified by the client. Errors for this RPC should be reported using the
+// https://github.com/googleapis/googleapis/blob/master/google/rpc/status.proto
+// message in the RPC return. The gnmi.Error message can be used to add additional
```

Comment:
maybe just clarify that `gnmi.Error` can be included via the protobuf.Any `details`  field